apis/server/image_bridge.go

```diff
@@ -53,6 +53,17 @@ func (s *Server) listImages(ctx context.Context, resp http.ResponseWriter, req *
 	return json.NewEncoder(resp).Encode(imageList)
 }
 
+func (s *Server) getImage(ctx context.Context, resp http.ResponseWriter, req *http.Request) error {
+	idOrRef := mux.Vars(req)["name"]
+
+	image, err := s.ImageMgr.GetImage(ctx, idOrRef)
+	if err != nil {
+		logrus.Errorf("failed to get image in containerd: %v", err)
```

Comment:
I do not think we should tell users details about `containerd` in pouch daemon's log. And there are others cases that errors are not from containerd.

Reply:
Updated. I also remove the `containerd` in listImages.